### DISCUSSION TITLE
Feature Suggestion

### DISCUSSION THREAD

**Discussion by henriqueleite42** [prisma/prisma#22109] [Q&A] [answered]
# Suggestion

Hey, I didn't find any other place to give feature suggestions, so I'm doing it here, but please tell me if there's any other place.

I think that would be awesome if Prisma had something like these projects:
- [Prisma Editor](https://prisma-editor.vercel.app/)
- [Prismaliser](https://prismaliser.app/)
- [Dbdocs](https://dbdocs.io/)

Somewhere that you can see all your relationships and have a nice documentation.

I know that [notiz-dev/prisma-dbml-generator](https://github.com/notiz-dev/prisma-dbml-generator) exists, but would be a lot nicer to use `schema.prisma` directly in an official source from Prisma instead of having to convert and host it elsewhere.

## Examples
![image](https://github.com/prisma/prisma/assets/50992814/c4302c74-d50c-4a88-b577-ff6a308dd906)

![image](https://github.com/prisma/prisma/assets/50992814/b15715c9-e095-464d-b577-7cca6d08a1d3)

**Comment by ludralph** [comment, marked as answer]
Hi @henriqueleite42 

Thank you for raising this.

I have converted this into a feature request as can be seen [here](https://github.com/prisma/prisma/issues/22123).

To make a feature request, you can always make use of this [link](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=)
